cpp/src/link_prediction/overlap_impl.cuh

```diff
@@ -27,19 +27,13 @@ struct overlap_functor_t {
   template <typename weight_t>
   weight_t __device__ compute_score(weight_t cardinality_a,
                                     weight_t cardinality_b,
-                                    weight_t cardinality_a_intersect_b) const
+                                    weight_t cardinality_a_intersect_b,
+                                    weight_t cardinality_a_union_b) const
   {
-    return cardinality_a_intersect_b / std::min(cardinality_a, cardinality_b);
-  }
-};
-
-struct weighted_overlap_functor_t {
-  template <typename weight_t>
-  weight_t __device__ compute_score(weight_t weight_a,
-                                    weight_t weight_b,
-                                    weight_t min_weight_a_intersect_b) const
-  {
-    return min_weight_a_intersect_b / std::min(weight_a, weight_b);
+    return (fabs(static_cast<double>(cardinality_a_union_b) - double{0}) <
+            double{2} / double{1 << 30})
```

Comment:
Same here, why are you subtracting 0 and what is `double{2} / double{1 << 30}`?

Reply:
That's correct, wasn't sure if I should use const from std namespace inside device functor.